src/containers/WelcomePage/WelcomePage.tsx

```diff
@@ -51,6 +52,222 @@ const getMultidisciplinaryTopics = (locale: LocaleType) => {
 const HiddenHeading = styled.h1`
   ${utils.visuallyHidden};
 `;
+export const programmeV2 = [
```

Comment:
Den her må du nesten flytte til en egen fil. vi har noe som ligner allerede, bytt heller ut den.

Reply:
Enig, men den skal nok ikke være her. Det er kun for å teste. Det er min feil skulle nevnt det

Reply:
Kanskje oppdatere filen istedenfor?